python/lsst/ts/audiotrigger/read_serial_temp_scanner.py

```diff
@@ -19,56 +19,77 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-__all__ = ["SerialTemperatureScanner"]
+__all__ = ["SerialTemperatureScanner", "execute_serial_temperature_scanner"]
 
+import argparse
 import asyncio
 import functools
 import logging
 from collections import OrderedDict
 
 import pigpio
-import serial
-from lsst.ts import tcpip
+from lsst.ts import tcpip, utils
+from lsst.ts.ess import common, controller
 
+from .constants import SENSOR_INDEX
 from .enums import Fan
-from .mocks import MockPio, MockThermalSensor
+from .mocks import MockPio
 
 
-class SerialTemperatureScanner(tcpip.OneClientServer):
+async def callback(data):
+    SerialTemperatureScanner.data = data
+
+
+def execute_serial_temperature_scanner():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--simulate", action="store_true")
```

Comment:
Are you sure you want this argument to default to `True`? That means that this needs to be started with `--simulate False` which is what we normally do not do. Please set the default value to `False`.